src/backend/dx12/src/conv.rs

```diff
@@ -470,7 +470,8 @@ pub fn map_image_flags(usage: image::Usage) -> D3D12_RESOURCE_FLAGS {
     use self::image::Usage;
     let mut flags = D3D12_RESOURCE_FLAG_NONE;
 
-    if usage.contains(Usage::COLOR_ATTACHMENT) {
+    // Blit operations implemented via a graphics pipeline
```

Comment:
actually, we should only do this if the format is renderable (yes, I'm missing this in Metal as well)

Reply:
We don't expose the image feature if not supported.